infrastructure/main.tf

```diff
@@ -13,9 +13,12 @@ locals {
   nonPreviewVaultName = "${var.raw_product}-${var.env}"
   vaultName           = "${local.is_preview ? local.previewVaultName : local.nonPreviewVaultName}"
   local_env           = "${local.is_preview ? "aat" : var.env}"
+  local_ase = "${(var.env == "preview" || var.env == "spreview") ? (var.env == "preview" ) ? "core-compute-aat" : "core-compute-saat" : local.ase_name}"
```

Comment:
this can be simplified to:
```
"${local.is_preview ? "core-compute-aat" : local.ase_name}"
```

spreview doesn't work so no point handling it

Reply:
Ive removed it and I think it might be ok ... how do I test this bit of the terraform?

Reply:
this isn't one also the formatting doesn't match up with how it should be
you need to run `terraform fmt`

Reply:
ok .. did not know that one.

Reply:
done

Reply:
the formatting in this file isn't right, terraform would normally aline all the = in block.
Please run `terraform fmt` from the infrastructure directory

Reply:
done ... seems there was a wierd crossover between intellij formatting terraform formatting.